bin/sirius-script-mamba-env-create.bash

```diff
@@ -0,0 +1,559 @@
+    printf_green "done!\n"
+else
+    printf_blue "already a member. Skipping...\n"
+fi
+
+cd /opt
+printf_yellow_clear "Create folder /opt/mamba_files for mamba installation: "
+if ! [ -d mamba_files ]
+then
+    sudo mkdir mamba_files
+    printf_green "done!\n"
+    printf_yellow_clear "Change permissions of folder /opt/mamba_files.\n"
+    sudo chown -R $USER:mamba mamba_files
+    chmod 774 -R mamba_files
+else
+    printf_blue "folder already exists. Skipping...\n"
+fi
+
+##############################################################################
+printf_yellow_clear "Install mamba in path /opt/mamba_files/mamba: "
+cd /opt/mamba_files
+if ! [ -d mamba ]
+then
+    wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-x86_64.sh
+    sh Mambaforge-Linux-x86_64.sh -b -p /opt/mamba_files/mamba
+    rm Mambaforge-Linux-x86_64.sh
+    printf_green "done!\n"
+else
+    printf_blue "there already is a mamba installation. Skipping...\n"
+fi
+
+printf_yellow_clear "Fix some folder permissions.\n"
+sudo chgrp -R mamba /opt/mamba_files/mamba
+sudo chmod 774 -R /opt/mamba_files/mamba
+sudo chown -R $USER.mamba ~/.conda
+
+printf_yellow_clear "Adding mamba and conda to path\n"
+source /opt/mamba_files/mamba/etc/profile.d/conda.sh
+source /opt/mamba_files/mamba/etc/profile.d/mamba.sh
+
+printf_yellow_clear "Adding mamba and conda paths to .bashrc: "
+cd ~/
+if ! grep -q "MAMBA_ADD" .bashrc;
+then
+    cat >> ~/.bashrc <<'EOM'
+# add conda and mamba to path
+MAMBA_ADD=/opt/mamba_files/mamba/etc/profile.d/mamba.sh
+if [ -f "$MAMBA_ADD" ] ; then
+    source "$MAMBA_ADD"
+fi
+CONDA_ADD=/opt/mamba_files/mamba/etc/profile.d/conda.sh
+if [ -f "$CONDA_ADD" ] ; then
+    source "$CONDA_ADD"
+fi
+EOM
+    printf_green "done!\n"
+else
+    printf_blue "paths already in ~/.bashrc file. Skipping...\n"
+fi
+
+##############################################################################
+printf_yellow "Create and prepare mamba enviroment\n"
+printf_yellow_clear "Create new mamba python environment named sirius: "
+if ! mamba env list | grep -q sirius
+then
+    mamba create --name sirius python=3.9.2 -y
+    printf_green "done!\n"
+else
+    printf_blue "environment already exists. Skipping...\n"
+fi
+
+### Activate Environment and do stuff
+printf_yellow_clear "Activate sirius enviroment\n"
+mamba activate sirius
+
+printf_yellow_clear "create link for python-sirius inside the enviroment: "
+cd $CONDA_PREFIX/bin
+if ! [ -f python-sirius ]
+then
+    ln -s python3 python-sirius
+    printf_green "done!\n"
+else
+    printf_blue "link alreay exists. Skipping...\n"
+fi
+
+printf_yellow "Install some mamba packages in sirius environment.\n"
+COMM="mamba install --freeze-installed -y"
+
+printf_yellow_clear "- First some system packages:\n"
+$COMM gxx make binutils swig build gsl libblas wmctrl fftw
+
+printf_yellow_clear "- Now some generic python packages:\n"
+$COMM pyparsing bottleneck aiohttp=3.7.4 scipy matplotlib pytest mpmath \
```

Comment:
for `aiohttp` must be `==3.7.4` to avoid installing 3.7.4.post0 version

Reply:
done!